res/home.html

```diff
@@ -123,6 +129,15 @@
         <div>
             <h1>_t("Welcome to Riot.im")</h1>
             <h2>_t("Decentralised, encrypted chat &amp; collaboration powered by [matrix]")</h2>
+            <p class="mx_HomePage_apps">_t("Riot.im available on all platforms"):
+              <a target="_blank" href="https://itunes.apple.com/us/app/vector.im/id1083446067"><img src="home/images/download_appstore.png" alt="Riot.im in AppStore"></a>
+              <a target="_blank" href="https://play.google.com/store/apps/details?id=im.vector.alpha"><img src="home/images/download_googleplay.png" alt="Riot.im in Google Play"></a>
+              <a target="_blank" href="https://f-droid.org/repository/browse/?fdid=im.vector.alpha"><img src="home/images/download_fdroid.png" alt="Riot.im in F-Droid"></a>
+              <a target="_blank" href="https://riot.im/download/desktop"><img src="home/images/download_windows.png" alt="Riot.im for Windows "></a>
+              <a target="_blank" href="https://riot.im/download/desktop"><img src="home/images/download_mac.png" alt="Riot.im for Mac"></a>
+              <a target="_blank" href="https://riot.im/download/desktop"><img src="home/images/download_linux.png" alt="Riot.im for Linux"></a>
+              <a target="_blank" href="https://riot.im/app/"><img src="home/images/download_browser.png" alt="Riot.im in any browser"></a>
```

Comment:
Isn't this page inside the riot itself? does it really make sense to link it?

Reply:
Our company use Riot on our domain like chat.ourcompany.org, and most of emploee don't know that Riot have standalone apps additionally to web interface in browser (before other employee tell about this). 
And I don't see other places in Riot interface with links to http://riot.im website or list of apps.
Those links will show them that Riot.im available not only via website.

Reply:
Also will be good to add link on `[matrix]` text in trailing h2 header, I have 2 variants:
```
            <h2>_t("Decentralised, encrypted chat &amp; collaboration powered by <a target="_blank" href=https://matrix.org/>[matrix]</a>")</h2>
```
```
            <h2>_t("Decentralised, encrypted chat &amp; collaboration powered by") <a target="_blank" href=https://matrix.org/>[matrix]</a></h2>
```
But both ways will change translation string, and all translations must be updated, so at now I don't add this in current PR.